src/field/field3d.cxx

```diff
@@ -375,6 +378,7 @@ Field3D & Field3D::operator=(const Field3D &rhs) {
 
   setLocation(rhs.location);
 
+  clearYupYdown();
```

Comment:
Would it be better to copy the yup/ydown (as we do for the data) in this case?

Reply:
> Would it be better to copy the yup/ydown (as we do for the data) in this case?

I don't know. I guess it only makes sense to copy yup/ydown if we also calculate yup/ydown in arithmetic operators. But if we calculate yup/ydown everywhere possible by default, we'd be doing a lot of computations that might not be needed (if we don't take a y-derivative, etc.). It would be nice to give an option for including yup/ydown fields, but I don't see a nice way to do it...